DashAI/back/api/api_v1/endpoints/runs.py

```diff
@@ -0,0 +1,207 @@
+import logging
+from typing import Union
+
+from fastapi import APIRouter, Depends, Response, status
+from fastapi.exceptions import HTTPException
+from sqlalchemy import exc, select
+from sqlalchemy.orm import Session
+
+from DashAI.back.api.deps import get_db
+from DashAI.back.database.models import Experiment, Run
+
+logging.basicConfig(level=logging.DEBUG)
+log = logging.getLogger(__name__)
+
+router = APIRouter()
+
+
+@router.get("/")
+async def get_run(run_id: int, db: Session = Depends(get_db)):
+    """
+    Returns the run with the spécified id.
+
+    Returns
+    -------
+    JSON
+        run JSON
```

Comment:
The docstring should indicate a python type (JSON is not) and a slightly more detailed description..

```suggestion
    dict
        A dict describing the requested run.
```